docs/fieldnorm/requirements.md

```diff
@@ -0,0 +1,20 @@
+# Requirements in software development
+The complexity and variety of developments in the field of bioinformatics, comes with a need for clear prerequisites before the start of a project. These requirements of software should describe the prerequisites that the software should meet and they should be clear before starting on a project. The requirements may relate to several aspects, including:
```

Comment:
"They should be clear before starting on a project" 

I would remove that, as that is almost never the case in practice. Very commonly when working on software it becomes clear that there are edge cases. This may effect the requirements. 
Also it may become clear that the initial requirements are not technically feasible. After discussion with the stakeholders it may be decided that the scope of the project is diminished to fit within the technical possibilities. 

Requiring requirements to be drilled down beforehand is unworkable. If the ISO norm does not mandate this, I would not be more stringent in the veldnorm.

Reply:
It is perfectly possible to adjust requirements when needed.  So yes I think requirements should be clear when starting a project, how can you otherwise start working on something if it is not clear.

Reply:
don't completely agree, requirements will change during the development. There should be starting requirements. Which should be as clear for a project to start... while developing it's important to discuss with stakeholders, and hence requirements might change. 
Requirements should be technically feasible.. otherwise there's no point having these requirements.. i.e. requirements should be written by both stakeholder and bioinformaticians.

Reply:
Maybe have a look at the "## 1.2.2 Requirements" section of the "Software Life Cycle Processes" subject. Here I have also tried to talk about this problem. Maybe you can use that bit to be more clear here (and create overlap between our subjects)